events/event_nettrace.go

```diff
@@ -15,6 +19,56 @@ var NetTraceIntegrator = func() func(basictracer.SpanEvent) {
 			tr = trace.New("tracing", t.OperationName)
 		case basictracer.EventFinish:
 			tr.Finish()
+		case basictracer.EventTag:
+			tr.LazyPrintf("%s:%v", t.Key, t.Value)
+		case basictracer.EventLogFields:
+			var buf bytes.Buffer
+
+			// Search for an "event" or "error" field for the "main" message.
```

Comment:
I can do that. But then we (cockroachdb) will postpone switching to the new KV api because it would make the messages messier.